src/core/style/mixins.scss

```diff
@@ -251,4 +251,21 @@
     cursor: default;
   }
 
+  &.md-indeterminate ._md-icon {
```

Comment:
@EladBezalel I could remove the `&`. The rule would still be valid and it would be one class less specific?